backend/routes.js

```diff
@@ -258,6 +267,21 @@ export default class Servers {
                         status = 201;
                     }
                     saveSetting('USERS', existingUsers);
+
+                    res.setCookie('plotly-auth-token', access_token, {'path': '/'});
+
+                    const db_connector_access_token = generateAndSaveAccessToken();
+                    res.setCookie('db-connector-auth-token', db_connector_access_token, {'maxAge': 300, 'path': '/'});
+                    res.setCookie('db-connector-user', username, {'path': '/'});
+
+                    if (that.electronWindow) {
+
+                        that.electronWindow.loadURL(`${protocol}://${domain}:${port}/`);
```

Comment:
:dolls: a comment here explaining that this redirect is done on the front-end for the browser version of this app might be helpful